src/helpers/surveyHelpers.tsx

```diff
@@ -11,3 +13,7 @@ export const combineLanguages = (
     .filter((langTitle) => langTitle?.length)
     .join(' / ');
 };
+
+export const getAnswerLabel = (lang: 'de' | 'pl', index: number) => {
+  return `${texts.survey.answerLabelPrefix[lang]} ${String.fromCharCode(65 + index)}:`;
```

Comment:
can i ask, what the idea behind the charcode stuff is behind the locale string, without making research on my own?

Reply:
it evaluates to the capital letter alphabet, based on the indices

Reply:
nice. could be a helper method or could have some comments so that i would have known directly whats going on while quick reading code.